extras/jenkins/DigitalOcean/Jenkinsfile

```diff
@@ -137,7 +137,7 @@ pipeline {
           $WORKSPACE/pulumi/python/venv/bin/pulumi config set docean:node_count "3" -C pulumi/python/config -s marajenkdo${BUILD_NUMBER}
           $WORKSPACE/pulumi/python/venv/bin/pulumi config set docean:region "sfo3" -C pulumi/python/config -s marajenkdo${BUILD_NUMBER}
           $WORKSPACE/pulumi/python/venv/bin/pulumi config set kic-helm:fqdn "mara${BUILD_NUMBER}.docean.mantawang.com" -C pulumi/python/config -s marajenkdo${BUILD_NUMBER}
-          $WORKSPACE/pulumi/python/venv/bin/pulumi config set docean:token "${DO_TOKEN}" --plaintext -C pulumi/python/config -s marajenkdo${BUILD_NUMBER}
+          $WORKSPACE/pulumi/python/venv/bin/pulumi config set digitalocean:token "${DO_TOKEN}" --plaintext -C pulumi/python/config -s marajenkdo${BUILD_NUMBER}
```

Comment:
**Question**:
So was the issue here that the pulumi namespace for digital ocean was `docean`?

Reply:
Partially; this is the correct namespace for the DO token in pulumi-land. However...it is still not working 100%, which I think is due to it not being encrypted hence the need for DIGITALOCEAN_TOKEN to be set in the environment. 

Since we have to rewrite things as part of the config refactoring I'm going to punt trying to make this encrypted down the road. It's not as simple as just adding in the encryption flag, since all the providers set their tokens through a superclass that I don't really want to fiddle with until we're ready for a rewrite.